### PR TITLE
Fix drawable for file

### PR DESCRIPTION
**Pull request by andela-wakinsanmi**

### CODE REVIEW

**Comment by smpete** [comment]
Can you explain this change? That's an incorrect asset for the attachment menu

**Comment by andela-wakinsanmi** [comment]
The asset we had was 70 X 80px, that's the reason it show too big. What I have done is to use Preview to resize it to appropriate size.

**Comment by smpete** [comment]
That asset looks incorrect anyways. Perhaps this will be fixed when `AND-1266` is fixed? Those added drawables are pixelated as well. Where did you get them?

**Comment by andela-wakinsanmi** [comment]
From the original one in drawable-xhdpi

**Comment by smpete** [comment]
We can't just scale up drawables to a larger size

File: layer-atlas/src/main/res/layout/ui_file_message_layout.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             android:layout_height="@dimen/layer_ui_file_message_image_height"
             android:adjustViewBounds="true"
             android:scaleType="center"
-            android:src="@drawable/ui_ic_file_generic"
+            android:src="@drawable/ic_insert_drive_file_white_24dp"
```
Comment (smpete) [inline layer-atlas/src/main/res/layout/ui_file_message_layout.xml]: This shouldn't change. This is used for the message preview, not the attachment menu.

File: layer-atlas/src/main/java/com/layer/ui/message/file/FileMessageModel.java [hunk 2]
```diff
@@ -167,7 +167,7 @@ public int getFileIconDrawable() {
     }
 
     private void setupFileIconDrawable(@Nullable String sourceMimeType) {
-        mFileIconDrawable = R.drawable.ui_ic_file_generic;
+        mFileIconDrawable = R.drawable.ic_insert_drive_file_white_24dp;
```
Comment (smpete) [inline layer-atlas/src/main/java/com/layer/ui/message/file/FileMessageModel.java]: Nor should this change. This is not in the attachment sender menu

**Pull request merged**